src/phpDocumentor/Descriptor/Interfaces/ConstantInterface.php

```diff
@@ -28,7 +28,9 @@ public function setTypes(Type $types) : void;
     /**
      * Returns the types that may be present in this constant.
      *
-     * @return Type[]
+     * @return list<?Type>
+     *
+     * @phpstan-return array<int, ?Type>
```

Comment:
Type is nullable? Again one to investigate later; I would not have expected this

Reply:
ConstantDescriptor have a getType method which checks against null and could theorically return null. I don't know if it happens, but if I don't document it here, I'll have to handle all the null cases in every child method to keep coherence.

Reply:
You did good; this just makes it painfully obvious that we need to fix something

Reply:
Made issue https://github.com/phpDocumentor/phpDocumentor/issues/2248 to resolve this